src/ui/MissingTerm/MissingTerms.ts

```diff
@@ -0,0 +1,219 @@
+import 'styling/_MissingTerms';
+import { exportGlobally } from '../../GlobalExports';
+import { Component } from '../Base/Component';
+import { IComponentBindings } from '../Base/ComponentBindings';
+import { ComponentOptions } from '../Base/ComponentOptions';
+import { $$, Initialization, l } from '../../Core';
+import { Dom } from '../../utils/Dom';
+import { analyticsActionCauseList, IAnalyticsIncludeMissingTerm } from '../Analytics/AnalyticsActionListMeta';
+import { IQueryResult } from '../../rest/QueryResult';
+import XRegExp = require('xregexp');
+
+export interface IMissingTermsOptions {
+  caption?: string;
+  clickable?: boolean;
+  numberOfTerms?: number;
+}
+
+/**
+ * This [result template component](https://docs.coveo.com/en/513/#using-result-template-components) renders a list of query terms
+ * that were not matched by its associated result item.
+ */
+export class MissingTerms extends Component {
+  static ID = 'MissingTerms';
+  static options: IMissingTermsOptions = {
+    /**
+     * Whether to allow the end-user to click a missing term to re-inject it in the query as
```

Comment:
to re-inject it in the query as an exact phrase match (i.e., as an expression between double quote characters). -> to filter out items that do not match this term.